docs/en_US/Tutorial/InstallCustomizedAlgos.rst

```diff
@@ -2,6 +2,16 @@
 **How to register customized algorithms as builtin tuners, assessors and advisors**
 =======================================================================================
 
+Content
+-------
+
+* `Overview`_
+* `Register customized algorithms as builtin tuners, assessors and advisors`_
+* `Use the installed builtin algorithms in experiment`_
+* `Manage builtin algorithms using nnictl algo`_
+* `Porting customized algorithms from v1.x to v2.x`_
+* `Example: Register a customized tuner as a builtin tuner`_
```

Comment:
no need to manually write this, you can simply use `.. contents::`

Reply:
fix it